main.cpp

```
@@ -1,14 +1,125 @@
+            default: // there is \ and after this nothing familier
+                char undefined_escape[1] = {lexema[i]};
+                output::errorUndefinedEscape(undefined_escape);
+                break;
+            }
+        }
+        else
+        {
+            buff += lexema[i];
+        }
+    }
+    output::printToken(yylineno, STRING, &buff[0]);
```

Cppcheck (2.10-2)

[error] Out of bounds access in expression 'buff[0]' because 'buff' is empty.

(containerOutOfBounds)